testLifecycleOperationsUser is flaky

It is very hard to get this test sometimes green. 
Example [test run](https://209.132.184.41:8493/logs/pull-266-20191219-205555-5ebcdca1-cockpit-project-cockpit-podman--fedora-31/log.html)
```
Traceback (most recent call last):
  File "test/check-application", line 441, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "test/check-application", line 498, in _testLifecycleOperations
    self.assertNotEqual(old_pid, new_pid)
AssertionError: '7412\n' == '7412\n'

not ok 5 testLifecycleOperationsUser (__main__.TestApplication) # duration: 44s
```

--- a/test/check-application
+++ b/test/check-application
@@ -494,8 +494,7 @@ class TestApplication(testlib.MachineCase):
         old_pid = self.execute(auth, "podman inspect --format '{{.State.Pid}}' swamped-crate")
         b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button:contains(Restart)')
         b.click('#containers-containers tbody tr:contains("busybox:latest") + tr ul.dropdown-menu li a:contains(Force Restart)')
-        new_pid = self.execute(auth, "podman inspect --format '{{.State.Pid}}' swamped-crate".strip())
-        self.assertNotEqual(old_pid, new_pid)
+        b.wait(lambda: old_pid != self.execute(auth, "podman inspect --format '{{.State.Pid}}' swamped-crate".strip()))
 
         with b.wait_timeout(5):
             self.check_container(container_sha, auth, ['swamped-crate', 'busybox:latest', 'sh', 'running'])

